spec/system/candidate_interface/continuous_applications/entering_details/candidate_can_not_edit_some_sections_after_first_submission_spec.rb

```diff
@@ -25,6 +26,13 @@
       given_i_already_have_one_submitted_application
       and_i_visit_your_details_page
       when_i_click_on_the_section_in_your_details_page
+      then_i_can_see_that_is_editable
```

Comment:
Is it worth checking that you can edit instead of just "seeing" since there's a test for not editing below?